src/ext/java/org/opentripplanner/ext/vectortiles/layers/stations/DigitransitStationPropertyMapper.java

```diff
@@ -15,23 +17,29 @@
 public class DigitransitStationPropertyMapper extends PropertyMapper<Station> {
 
   private final TransitService transitService;
+  private final Locale locale;
 
-  private DigitransitStationPropertyMapper(TransitService transitService) {
+  private DigitransitStationPropertyMapper(TransitService transitService, Locale locale) {
     this.transitService = transitService;
+    this.locale = locale;
   }
 
-  public static DigitransitStationPropertyMapper create(TransitService transitService) {
-    return new DigitransitStationPropertyMapper(transitService);
+  public static DigitransitStationPropertyMapper create(
+    TransitService transitService,
+    Locale locale
+  ) {
+    return new DigitransitStationPropertyMapper(transitService, locale);
   }
 
   @Override
   public Collection<T2<String, Object>> map(Station station) {
     var childStops = station.getChildStops();
 
+    var name = station.getName().toString(locale);
+
     return List.of(
       new T2<>("gtfsId", station.getId().toString()),
-      // Name is I18NString now, we return default name
-      new T2<>("name", station.getName().toString()),
+      new T2<>("name", station.getName().toString(locale)),
```

Comment:
Use `I18NStringMapper`?

Reply:
I thought about it but it just adds a null check and the station name is non null. Does it still make sense?

Reply:
I created another mapper in https://github.com/opentripplanner/OpenTripPlanner/pull/4529/commits/e023e708ad97d408788777335bf1ba89b1f906b4